### PR TITLE
Terraform updated to 0.9.11

### PR DESCRIPTION
**Pull request by anmolnagpal**

### CODE REVIEW

File: terraform/Dockerfile-light [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM alpine:latest
 MAINTAINER "HashiCorp Terraform Team <terraform@hashicorp.com>"
 
-ENV TERRAFORM_VERSION=0.9.10
+ENV TERRAFORM_VERSION=0.9.11
 ENV TERRAFORM_SHA256SUM=77f0d01182d665f7f3c63c326aa699b452fba043c2e2f9050c4bd114f98a1207
```
Comment (analytically) [inline terraform/Dockerfile-light]: this won't be the same

Comment (anmolnagpal) [inline terraform/Dockerfile-light]: @analytically  updated

**Comment by SmoshySmosh** [comment]
+1 can we get this shipped?

**Comment by grahamlyons** [comment]
Is there any reason not to merge this? Is there anything we can help with to speed up getting the latest Terraform on Docker Hub?

**Comment by stack72** [comment]
LGTM - thanks for this

**Comment by SmoshySmosh** [comment]
@stack72 Dockerhub was not updated with the latest.

**Comment by catsby** [comment]
https://github.com/hashicorp/docker-hub-images/issues/41 is also tracking this

**Pull request merged**